precompiles/collective/Collective.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.0;
```

Comment:
```suggestion
pragma solidity >=0.8.3;
```

Reply:
Any reason it should be bumped? Our other precompiles use 0.8.0 too.

Reply:
Since #1754 our precompiles requires 0.8.3

Reply:
We bumped it in https://github.com/PureStake/moonbeam/pull/1754 due to a linting issue reported in `0.8.0`